lib/postProcessing/filter.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 var filter = module.exports = { };
 
+var _ = require("underscore");
 var debug = require("../debugging.js");
 
 filter.action = function(request, response, callback) {
-  var allFilters = request.params.filter;
+  var allFilters = _.extend({ }, request.params.filter);
```

Comment:
`_.clone(request.params.filter)` should achieve the same result for this case and read better. What do you think?